src/lib/coap/sol-oic-server.c

```diff
@@ -698,9 +701,21 @@ _sol_oic_resource_type_handle(
         return -1;
     }
 
-    if (!handle_fn) {
-        code = SOL_COAP_RSPCODE_NOT_IMPLEMENTED;
+    if (sol_coap_packet_get_payload(response, &response_payload, &response_len) < 0)
         goto done;
+
+    r = strlen(response_prefix);
```

Comment:
Makes no difference here since these are (or should be) `static const`, so the compiler will optimize these out.